cli/index_test.ts

```diff
@@ -2,22 +2,17 @@
 import { expect } from "chai";
 import * as fs from "fs-extra";
 import { dump as dumpYaml, load as loadYaml } from "js-yaml";
-import * as os from "os";
 import * as path from "path";
 
-import { ChildProcess, execFile } from "child_process";
+import { execFile } from "child_process";
 import { version } from "df/core/version";
 import { dataform } from "df/protos/ts";
-import { suite, test } from "df/testing";
+import { suite, test, getProcessResult, nodePath, npmPath, corePackageTarPath } from "df/testing";
```

Comment:
import should be alphabetized. same in other locations.

Reply:
Let's not bother if the linter won't do it, and it doesn't look there's a drop-in setting

Reply:
The linter was throwing, I somehow merged it before the tests finished lol - will update in subsequent PR